src/main/java/appeng/core/api/definitions/ApiBlocks.java

```diff
@@ -476,8 +479,7 @@ private static IBlockDefinition makeStairs( String registryName, FeatureFactory
 	{
 		return registry.block( registryName, () -> new BlockStairCommon( block.maybeBlock().get(), block.identifier() ) )
 				.features( AEFeature.DecorativeQuartzBlocks )
-				.rendering( new BlockRenderingCustomizer()
-				{
+				.rendering( new BlockRenderingCustomizer(){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)